src/zabapgit_services.prog.abap

```diff
@@ -2,8 +2,9 @@
 *&  Include           ZABAPGIT_SERVICES
 *&---------------------------------------------------------------------*
 
+INCLUDE zabapgit_services_git.
+INCLUDE zabapgit_services_trans_branch.
```

Comment:
How about `zabapgit_trans_utils` ? But any would agree to anything without 'services' in the name :) And also it should be moved to main prog.